tools/build_docs.py

```diff
@@ -36,11 +36,36 @@ def get_theme(path: Path, url: str):
         check_call(tar_cmd)
 
 
-def main():
+def update_release_notes(version):
+    """Gather newsfragments and add them to the release notes.
+
+    Args:
+        version (str):
+            Version to set the section to for all newsfragments. Newsfragments will be
+            added to the release notes and the now old newsfragments will be staged for
+            removal.
     """
-    Build documentation/website
+    print(f"Adding newsfragment enteries to release notes for release {version}")
+    check_call(shlex.split("towncrier build --version {version} --yes"))
+
+
+def main(version=None):
```

Comment:
@LarsAsplund where in the repo does it define the path to the documentation to upload to RTD? I've looked around for a while and haven't been able to figure this out. Do you manually upload the docs to RTD after a release?

In essence, I need to modify the sphinx call here to define a different location to save the release version of the docs as I am getting errors when running the release script with the sys.argv since I now have to call this main func in release.py.

Reply:
It seems to me like it is expected that the tox py*-docs env is run before release.py, but I'm not seeing where a script or GHA is actually pointing to the HTML to upload to RTD.

Reply:
This action needs to be modified: https://github.com/VUnit/vunit/blob/master/.github/workflows/docs.yml

Reply:
`tox py*-docs` generates the HTML output in https://github.com/VUnit/vunit/blob/master/.github/workflows/docs.yml#L45. The website is updated by creating an orphan branch, adding the content and pushing it to repo vunit.github.io using a private SSH key: https://github.com/VUnit/vunit/blob/master/.github/workflows/docs.yml#L48-L61.

Docs are updated after each push to master, regardless of it being a release: https://github.com/VUnit/vunit/blob/master/.github/workflows/docs.yml#L41.